lavorare-su-nebulab/blog.md

```diff
@@ -0,0 +1,35 @@
+# Blog
+
+[Il nostro blog](https://nebulab.it/blog/) è gestito da Shana ed è uno dei modi con cui contribuiamo 
+al panorama OSS, nonché un modo per farci conoscere da potenziali clienti. Chiunque può scrivere sul 
+blog e tutti sono incoraggiati a farlo! Scrivere è anche un ottimo modo per capire gli argomenti che 
+stiamo studiando e i problemi che stiamo affrontando, dunque il risutlato di un progetto del venerdì 
+potrebbe essere un articolo sul blog.
+
+## Scrivere un post
+
+Scrivere un post è un processo che si compone di quattro fasi ed è completamente gestito in un
+[progetto GitHub](https://github.com/nebulab/nebulab/projects/3):
+
+1. **Proposta:** se vuoi proporre un articolo, apri una issue e aggiungila al backlog del progetto 
```

Comment:
Attualmente questo cambia il nostro flusso che prevede o proposta spontanea o scrittura di articoli su proposta di Shana per esigenze SEO. Attualmente non funziona quindi, se scrivere su iniziativa spontanea pensiate risolva, per me avete un bel pollicione all'in su 👍

Reply:
@matteogalliani penso sia una buona idea, ma aggiungerò che volendo puoi anche prendere articoli che già sono stati proposti da Shana.